PushNotifications.md

```diff
@@ -236,17 +234,93 @@ final message = ably.Message(
 
 ### Receiving Messages
 
+To see an example of receiving messages and handling other notification related behaviour, look at [push_notification_handlers](example/lib/push_notifications/push_notification_handlers.dart) in the example app.
+
 #### Notification Message / Alert Push Notification
 
-**Android**: You cannot handle notification messages as they are shown to the user without calling any methods in your application. To create notifications which launch the application to a certain page (notifications which contain deep links or app links), or notifications which contain buttons/actions, images, and inline replies, you should send a data message and create a notification when the message is received.
+**Android**: In the case where the app is in the background / terminated, you cannot configure / disable notification messages as they are automatically shown to the user by Firebase Messaging Android SDK. To create notifications which launch the application to a certain page (notifications which contain deep links or app links), or notifications which contain buttons / actions, images, and inline replies, you should send a data message and create a notification when the message is received. 
+
+**iOS**: In the case where the app is in the background / terminated, ably-flutter doesn't provide the functionality to configure / extend alert notifications on iOS, and these will automatically be shown to the user.
```

Comment:
In the case where -> When

Library names should be in code font throughout (`ably-flutter`)